tool/grind.dart

```diff
@@ -67,11 +66,11 @@ build() {
   log('${mainFile} compiled to ${_printSize(embedFile)}');
 
   // Delete the build/web/packages directory.
-  delete(getDir('build/web/packages'));
+  // delete(getDir('build/web/packages'));
```

Comment:
I think we can delete the commented out code here (and below, w/ the `reify the symlinks`).